### ISSUE TITLE
start error

### ISSUE THREAD

**Issue by jiajiayang** [telepresenceio/telepresence#1308]
### What were you trying to do?

(please tell us)

### What did you expect to happen?

(please tell us)

### What happened instead?

(please tell us - the traceback is automatically included, see below.
use https://gist.github.com to pass along full telepresence.log)

### Automatically included information

Command line: `['/usr/local/bin/telepresence']`
Version: `0.104`
Python version: `3.7.7 (default, Mar 10 2020, 15:43:33) 
[Clang 11.0.0 (clang-1100.0.33.17)]`
kubectl version: `Client Version: v1.18.1 // Server Version: v1.16.2`
oc version: `(error: [Errno 2] No such file or directory: 'oc': 'oc')`
OS: `Darwin Lys-MacBook-Pro.local 19.4.0 Darwin Kernel Version 19.4.0: Wed Mar  4 22:28:40 PST 2020; root:xnu-6153.101.6~15/RELEASE_X86_64 x86_64`

```
Traceback (most recent call last):
  File "/usr/local/bin/telepresence/telepresence/cli.py", line 135, in crash_reporting
    yield
  File "/usr/local/bin/telepresence/telepresence/main.py", line 65, in main
    remote_info = start_proxy(runner)
  File "/usr/local/bin/telepresence/telepresence/proxy/__init__.py", line 138, in start_proxy
    run_id=run_id,
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 144, in get_remote_info
    runner, deployment_name, deployment_type, run_id=run_id
  File "/usr/local/bin/telepresence/telepresence/proxy/remote.py", line 91, in get_deployment_json
    return json.loads(output)["items"][0]
IndexError: list index out of range

```

Logs:

```
ound svc,deploy --selector=telepresence=5d81582fb03941748eab4e0e2f00d926
   7.1  10 | No resources found
   7.1 TEL | [10] ran in 0.27 secs.
   7.1 TEL | [11] Running: kubectl --context context-cluster1-admin --namespace default run --restart=Always --limits=cpu=1000m,memory=256Mi --requests=cpu=25m,memory=64Mi telepresence-1587085328-90591-14345 --image=datawire/telepresence-k8s:0.104 --labels=telepresence=5d81582fb03941748eab4e0e2f00d926
   7.3  11 | pod/telepresence-1587085328-90591-14345 created
   7.3 TEL | [11] ran in 0.23 secs.
   7.3 TEL | END SPAN deployment.py:108(create_new_deployment)    0.5s
   7.3 TEL | BEGIN SPAN remote.py:142(get_remote_info)
   7.3 TEL | BEGIN SPAN remote.py:75(get_deployment_json)
   7.3 TEL | [12] Capturing: kubectl --context context-cluster1-admin --namespace default get deployment -o json --selector=telepresence=5d81582fb03941748eab4e0e2f00d926
   7.6 TEL | [12] captured in 0.22 secs.
   7.6 TEL | END SPAN remote.py:75(get_deployment_json)    0.2s

```

**Comment by marun** [comment]
Duplicate of https://github.com/telepresenceio/telepresence/issues/1297

**Comment by marun** [comment]
Duplicate of #1297, which was fixed by #1312. Should be resolved in the next release, and until then #1297 documents a number of workarounds.
